app/main.py

```diff
@@ -1,6 +1,31 @@
+import os
+import requests
+
+API_KEY = os.environ["API_KEY"]
+
+CITY = os.environ.get("CITY", "Paris")
+URL = (
+    f"http://api.openweathermap.org/data/2.5/weather"
+    f"?q={CITY}"
+    f"&appid={API_KEY}"
+    f"&units=metric"
+)
```

Comment:
Code Style Issue: According to the checklist, it is recommended to use constants for URLs. In this case, 'http://api.openweathermap.org/data/2.5/weather' is a constant value and should be defined as a constant at the top of your file.